azurelinuxagent/ga/update.py

```diff
@@ -312,7 +315,7 @@ def _emit_restart_event(self):
         self._set_sentinal() 
         return
 
-    def _ensure_latest_agent(self, base_version=CURRENT_VERSION):
+    def _upgrade_available(self, base_version=CURRENT_VERSION):
```

Comment:
I found it a little confusing :) once I changed the name it seemed more readable to me..